.github/ISSUE_TEMPLATE/release-team-lead-progress.md

```diff
@@ -0,0 +1,271 @@
+    - Add an `approvers` entry in `releases/release-x.y/OWNERS`
+  - **Release Team Lead only**
+    - In `OWNERS_ALIASES`, add an entry in the following sections:
+      - `release-team`
+      - `release-team-lead-role`
+- [ ] Updated Google Groups/GCP IAM membership [(`kubernetes/k8s.io`)](https://git.k8s.io/k8s.io/groups/groups.yaml)
+  - `k8s-infra-release-viewers@`
+  - `release-managers@`
+  - `release-team@`
+- [ ] Release Team calendar access granted (reach out to sig-release chairs)
+- [ ] Zoom credentials (host key) granted (reach out to sig-release chairs)
+- [ ] Added incoming release team leads to `release-team-leads` Slack Group [(`kubernetes/community`)](https://git.k8s.io/community/communication/slack-config/sig-release/usergroups.yaml)
+  - Add slack ID(s) to [`users.yaml`](https://git.k8s.io/community/communication/slack-config/users.yaml), if they are not yet in the file
+  - Add username(s) to [`usergroups.yaml`](https://git.k8s.io/community/communication/slack-config/sig-release/usergroups.yaml)
+- [ ] Ensured SIG Release co-chairs replaced the outgoing Emeritus Advisor (EA) with the new EA as an owner of the [kubernetes-release-team-shadows](https://groups.google.com/a/kubernetes.io/g/release-team-shadows) Google Group.
+- [ ] Ensured top-level `OWNERS_ALIASES` only includes Release Team personnel from four (4) releases, including the current one.
+
+**Create Release Team Documents:**
+- [ ] Meeting Notes document created
+  - Access: public comment access, edit access [kubernetes-release-team](https://groups.google.com/a/kubernetes.io/g/release-team)
+  - Bit.ly: `bit.ly/k8s1XX-releasemtg`
+  - Example: [1.26](https://bit.ly/k8s126-releasemtg)
+- [ ] Retro document created
+  - Access: public comment access, edit rights shared with [kubernetes-release-team](https://groups.google.com/a/kubernetes.io/g/release-team)
+  - Bit.ly: `bit.ly/k8s1XX-retro`
+  - Example: [1.26](https://bit.ly/k8s126-retro)
+- [ ] Release Team contact sheet created
+  - Access: **restricted access** edit rights shared with [kubernetes-release-team](https://groups.google.com/a/kubernetes.io/g/release-team)
+  - Bit.ly: `bit.ly/k8s1XX-contacts`
+- [ ] Lead rotation sheet created
+  - Access: : **restricted access**, edit rights shared with release team lead shadows individually
+
+**One week before the start of the release cycle:**
+- [ ] "Release Sneak Peak" email to introduce yourself, lead shadows, role leads, and branch manager has been send out
+  - Send the email to [k-dev](https://groups.google.com/a/kubernetes.io/g/dev), [SIG Leads](https://groups.google.com/a/kubernetes.io/g/leads), [SIG Release](https://groups.google.com/forum/#!forum/kubernetes-sig-release), [Release Team](https://groups.google.com/a/kubernetes.io/g/release-team)
+  - Example: [1.24 sneak peek](https://groups.google.com/a/kubernetes.io/g/release-team/c/hePU9eaoYEw), [1.25 sneak peek](https://groups.google.com/a/kubernetes.io/g/release-team/c/jwqlcrctFO0)
```

Comment:
These examples are on the release-team mailing list, so they don't work if you're not already on the team. Change to the k-dev versions?

Reply:
yes, good catch!